### PR TITLE
Add the story of MonkeyKing: Hero is Back

### PR DESCRIPTION
**Pull request by cakewong**
see the part of story[here](https://github.com/cakewong/create-your-own-adventure/blob/master/Chinese/MonkeyKingHeroisBack/MonkeyKingHeroIsBack.md).
Or search it with[百度](https://www.baidu.com/s?wd=%E5%A4%A7%E5%9C%A3%E5%BD%92%E6%9D%A5&rsv_spt=1&issp=1&f=8&rsv_bp=0&rsv_idx=2&ie=utf-8&tn=63090008_1_hao_pg&rsv_enter=1&rsv_sug3=7&rsv_sug1=1)

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2338)

### CODE REVIEW

File: Chinese/大话西游.md [hunk 1]
```diff
@@ -8,42 +8,44 @@
...
-[等死朕了](howeroc/西游后记.md)
+[蝑香��](howeroc/镼踵虜�扇.md)
 
-[jubeater](ZTL/无天.md)
+[jubeater](ZTL/�予.md)
 
-[��ɽ����](heishanlaoyao/heishanlaoyao.md)?
+[窪刓橾毦](heishanlaoyao/heishanlaoyao.md)?
+
+[MonekyKing:Hero is Back](https://github.com/cakewong/create-your-own-adventure/blob/master/Chinese/MonkeyKingHeroisBack/MonkeyKingHeroIsBack.md)
```
Comment (cakewong) [inline Chinese/大话西游.md]: the link seems weird, you better check it

**Comment by cakewong** [comment]
I modified the 大话西游.md, make my story's link more standard. I checked it. please agree with mine

**Comment by cakewong** [comment]
update my homework,

**Comment by cbuckey-uda** [comment]
Sorry about the delay, @cakewong! @casey-collab, please review this.

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Comment by cbuckey-uda** [comment]
@cakewong, the "deleted" lines are fine. But I can't merge this because it conflicts with another change. If you resolve the conflicts, I'll be happy to merge this.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**